File: tests/Generator/resource/swift/complex/Argument.swift
// Describes arguments of the operation
class Argument: Codable {
    var contentType: String?
    var _in: String?
    var name: String?
    var schema: PropertyType?

    enum CodingKeys: String, CodingKey {
        case contentType = "contentType"
        case _in = "in"
        case name = "name"
        case schema = "schema"
    }
}

File: tests/Generator/resource/kotlin/complex/Argument.kt

import com.fasterxml.jackson.annotation.*

/**
 * Describes arguments of the operation
 */
open class Argument {
    @JsonProperty("contentType") var contentType: String? = null
    @JsonProperty("in") var in: String? = null
    @JsonProperty("name") var name: String? = null
    @JsonProperty("schema") var schema: PropertyType? = null
}

